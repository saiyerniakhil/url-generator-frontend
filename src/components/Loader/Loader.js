import "./index.css";
const Loader = () => {
  return (
    <>
      <div className="loadingio-spinner-eclipse-5av0r8f4i1c">
        <div className="ldio-wr4laxx1k9">
          <div></div>
        </div>
      </div>
    </>
  );
};

export default Loader;
