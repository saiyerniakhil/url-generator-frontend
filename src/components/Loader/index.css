@keyframes ldio-wr4laxx1k9 {
  0% {
    transform: rotate(0deg);
  }
  50% {
    transform: rotate(180deg);
  }
  100% {
    transform: rotate(360deg);
  }
}
.ldio-wr4laxx1k9 div {
  position: absolute;
  animation: ldio-wr4laxx1k9 1s linear infinite;
  width: 160px;
  height: 160px;
  top: 20px;
  left: 20px;
  border-radius: 50%;
  box-shadow: 0 4px 0 0 #007bff;
  transform-origin: 80px 82px;
}
.loadingio-spinner-eclipse-5av0r8f4i1c {
  width: 200px;
  height: 200px;
  display: inline-block;
  overflow: hidden;
  background: #ffffff;
}
.ldio-wr4laxx1k9 {
  width: 100%;
  height: 100%;
  position: relative;
  transform: translateZ(0) scale(1);
  backface-visibility: hidden;
  transform-origin: 0 0; /* see note above */
}
.ldio-wr4laxx1k9 div {
  box-sizing: content-box;
}
/* generated by https://loading.io/ */
