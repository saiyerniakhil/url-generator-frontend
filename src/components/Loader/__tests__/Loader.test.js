import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Loader from '../Loader';

test("Loader renders properly", () => {
    expect(<Loader/>).toMatchSnapshot()
})