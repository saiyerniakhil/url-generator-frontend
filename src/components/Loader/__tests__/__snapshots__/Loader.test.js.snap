// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders properly 1`] = `<Loader />`;
